src/platform-includes/getting-started-install/javascript.remix.mdx

```diff
@@ -1,7 +1,25 @@
-```bash {tabTitle:npm}
-npm install --save @sentry/remix
-```
+We recommend installing the SDK through our installation wizard:
 
-```bash {tabTitle:Yarn}
-yarn add @sentry/remix
+```bash
+npx @sentry/wizard@latest -i remix
 ```
+
+The wizard will prompt you to log in to Sentry. It will then automatically do the following steps for you:
+
+- create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't exist yet)
+- in these files, add your initialization code for the client-side SDK and server-side SDK, respectively.
+- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
+- add an example page to your app to verify your Sentry setup
+
+The two configuration types are mostly the same, except that some configuration features, like Session Replay, only work in `entry.client.tsx`.
```

Comment:
The main instrumentation work is done outside of `entry` files. So, might be better to mention v2 flags here instead.

```suggestion
If you use [Remix future flags](https://remix.run/docs/en/main/pages/api-development-strategy#current-future-flags), the wizard will instrument your application accordingly to support Remix v2 features. 
```